font-codegen/src/parsing.rs

```diff
@@ -235,6 +235,12 @@ pub(crate) struct FieldReadArgs {
     pub(crate) inputs: Vec<syn::Ident>,
 }
 
+#[derive(Clone, Debug)]
+pub(crate) struct Available {
```

Comment:
Nit: I didn't find the meaning of available intuitive. Maybe it's worth the cost of a long name? - for example available_from_version(2) I would immediately get

Reply:
happy to bikeshed that name, I agree it isn't great. If we agree on a change will do a separate PR. :)